care/users/api/viewsets/change_password.py

```diff
@@ -48,3 +67,11 @@ def update(self, request, *args, **kwargs):
             return Response({"message": "Password updated successfully"})
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    def has_permission(self, request, user):
+        authuser = request.user
+        return (
+            authuser == user
+            or authuser.is_superuser
+            or authuser.user_type >= User.TYPE_VALUE_MAP["DistrictAdmin"]
```

Comment:
This is not required, lets only allow it for superadmins. we are going to remove the dist and state admins soon. 
also a dist admin only has access to a user if he is of the same dist.